cg/cli/downsample.py

```diff
@@ -0,0 +1,74 @@
+"""cg module for downsampling reads in a sample."""
+
+import logging
+from pathlib import Path
+from typing import Tuple
+
+import click
+
+from cg.apps.downsample.downsample import DownsampleAPI
+from cg.apps.downsample.utils import store_downsampled_sample_bundle
+from cg.constants.constants import DRY_RUN
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def downsample():
+    """Downsample reads in a sample."""
+
+
+@downsample.command("samples", help="Downsample reads in one or multiple samples in a case.")
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="Case identifier used in statusdb, e.g. supersonicturtle. The case information wil be transferred.",
```

Comment:
So - the case id given here does not need to be connected to the samples. My thoughts then are - is it correct to say that the case is a downsampled version of the old one if it is not the same? I think that we should refer to adding the `<case_id>_downsampled` name to the case, and instead just give it a new pet name. The samples themselves that are connected to the case keep that information.